packages/core/package.json

```diff
@@ -65,7 +66,8 @@
 		"@babel/preset-env": "^7.22.20",
 		"@babel/preset-typescript": "^7.22.15",
 		"@types/elliptic": "^6.4.14",
-		"@types/sha.js": "^2.4.1"
+		"@types/sha.js": "^2.4.1",
+		"long5": "npm:long"
```

Comment:
why is this aliased as `long5`?

Reply:
Was running into some weird issues where we import 2 different types of Long using 'import Long from 'long';
There's [@types/long](https://www.npmjs.com/package/@types/long) that's used by one of the interfaces in cosmjs-types and [long.js](https://github.com/dcodeIO/Long.js/) that's used by the ScheduledTokenReleaseSDKType created by our proto library.

The 2 types don't seem to be substitutable for each other so I added an alias for long.js